stl/inc/bitset

```diff
@@ -164,6 +164,36 @@ public:
         _Construct<_Traits>(_Str.data() + _Pos, _Count, _Elem0, _Elem1);
     }
 
+#ifdef _HAS_CXX26
+    template <class _Elem>
+    // currently no _CONSTEXPR26, it will be added at the same time with _HAS_CXX26
+    /*_CONSTEXPR26 */ inline explicit bitset(const _Elem* _Ntcts,

```

Comment:
Given these constructors are guarded by `_HAS_CXX26`, we can use unconditional `constexpr`.

Reply:
Oh, `#if _HAS_CXX26` should be used here. In older modes it will be defined as `0`.

Reply:
WG21-P2697R1 only adds one constructor to `std::bitset`.

The old constructor is actually unchanged, because `basic_string_view<C>::size_type` and `basic_string<C>::size_type` are both `size_t` (see [[string.view.template.general]](https://eel.is/c++draft/string.view.template.general), [[basic.string.general]](https://eel.is/c++draft/basic.string.general), and [[default.allocator.general]](https://eel.is/c++draft/default.allocator.general)).